### ISSUE TITLE
Ordered and unordered lists

### ISSUE THREAD

**Issue by pablox-cl** [robotlolita/raven#95]
I expected to being able to write

```
+ buy milk
+ read book
```

Like on markdown, and have them transform to a beautiful list, but it doesn't work (I tried with `+`, `*` and `-`). I don't know if it's not possible, or I haven't found how :P.

Thanks for the great app, I'm loving it

**Comment by robotlolita** [comment]
Raven doesn't support markdown right now ([it's on the roadmap, though](https://github.com/robotlolita/raven/blob/master/ROADMAP.org#allow-formatting-text-with-markdown)). I'm not sure lists make much sense in a text editor aimed at writing novels though.

**Comment by pablox-cl** [comment]
Yeah, I know... I understand.... but they could be useful when creating dialogues, don't you think? :)
